eclass/kernel-build.eclass

```diff
@@ -234,6 +242,12 @@ kernel-build_src_compile() {
 	debug-print-function ${FUNCNAME} "${@}"
 
 	emake O="${WORKDIR}"/build "${MAKEARGS[@]}" all
+
+	# Now that we have a relfile, set KV_FULL
+	if [[ -z ${KV_FULL} ]]; then
+		local relfile=${WORKDIR}/build/include/config/kernel.release
+		KV_FULL=$(<"${relfile}") || die
+	fi
```

Comment:
Can KV_FULL ever be different from kernel.release? If not, add a sanity check.

Reply:
This is checked in `kernel-install_pkg_preinst()` already. do we need to replicate that check here? I think given that, and the fact that there's no _real_ use case for overriding it from the ebuild, we could just unconditionally set it here and where relevant in `kernel-install.eclass`. That would also resolve the ambiguous `@DESCRIPTION`s

Reply:
Hmm, but `kernel-install.eclass` may actually require setting it in ebuild, if it's not `${PV}...`, correct?

My point is mostly that we want to fail early.

Reply:
I have moved the setting of `KV_FULL` into `kernel-build_src_configure()`, which means we can also check it for sanity at that step, before we go through the rigmarole of building the entire kernel.